sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslSqlStdOperatorsTest.java

```diff
@@ -193,6 +193,29 @@ public void testThatOperatorsExist() {
     }
   }
 
+  @Test
+  @SqlOperatorTest(name = "<", kind = "LESS_THAN")
```

Comment:
Can you leave these lines here, but comment them out? Here is why: when you have an annotation like this it tells the framework that we already have enough tests for this operator. It is great that you added some tests, but we should get a whole suite for the other types too before we say we are done.

Reply:
got it. I have seen #5744 adding tests for comparison operator. I can append my tests in this test case.